### PR TITLE
Adds the fleet officer coveralls to the loadout

### PR DESCRIPTION
**Pull request by JohnBigless**
Adds the fleet officer coveralls as an NTEF only, Command only uniform.
Also adds a few loadout defines. Check the code yourself, i can't be bothered to write them all here.

### CODE REVIEW

File: maps/torch/loadout/_defines.dm [hunk 1]
```diff
@@ -48,6 +48,9 @@
 //For jobs that spawn with armor in their lockers
 #define ARMORED_ROLES list(/datum/job/captain, /datum/job/hop, /datum/job/rd, /datum/job/cmo, /datum/job/chief_engineer, /datum/job/hos, /datum/job/qm, /datum/job/sea, /datum/job/sea/marine, /datum/job/bridgeofficer, /datum/job/officer, /datum/job/warden, /datum/job/detective, /datum/job/merchant, /datum/job/bodyguard, /datum/job/bailiff, /datum/job/submap/skrellscoutship_crew, /datum/job/submap/skrellscoutship_crew/leader, /datum/job/grunt, /datum/job/combat_tech, /datum/job/squad_lead, /datum/job/seccadet)
 
+//For members of the command and command support department. Why wasn't this here before?
+#define COMMAND_ROLES list(/datum/job/captain, /datum/job/hop, /datum/job/rd, /datum/job/cmo, /datum/job/chief_engineer, /datum/job/hos, /datum/job/sea, /datum/job/sea/marine, /datum/job/bridgeofficer, /datum/job/liaison, /datum/job/bodyguard, /datum/job/bailiff, /datum/job/psiadvisor, /datum/job/adjudicator)
```
Comment (Roland410) [inline maps/torch/loadout/_defines.dm:52]: This probably should stay as command only then. Don't think the Adjudicator, who is a civvie would qualify and generally stuff that applies to heads won't apply to the CL, APA etc.

File: maps/torch/loadout/loadout_uniform_boh.dm [hunk 2]
```diff
@@ -18,3 +18,11 @@
 	path = /obj/item/clothing/under/solgov/utility/fleet
 	cost = 0
 	allowed_branches = NT_BRANCHES
+
+/datum/gear/uniform/fleet/officer
+	display_name = "officer fleet fatigue"
```
Comment (Roland410) [inline maps/torch/loadout/loadout_uniform_boh.dm:23]: Fleet officer fatigues instead maybe? Sounds better to me.

File: maps/torch/loadout/_defines.dm (hunk 1 above)
Comment (JohnBigless) [reply maps/torch/loadout/_defines.dm:52]: I thought that the Adjucator, Corporate Liason and Asset Protection Agent is *technically* a member of Command Support. Do color me wrong, though.

File: maps/torch/loadout/loadout_uniform_boh.dm (hunk 2 above)
Comment (JohnBigless) [reply maps/torch/loadout/loadout_uniform_boh.dm:23]: Tried to follow the original 'naming method', but ok.

File: maps/torch/loadout/_defines.dm (hunk 1 above)
Comment (Roland410) [reply maps/torch/loadout/_defines.dm:52]: They are indeed command support, but that's often used in a more broad sense. Technically the way you set up the uniform it shouldn't cause issues though.

Comment (JohnBigless) [reply maps/torch/loadout/_defines.dm:52]: The new item that uses this new `#define` is also restricted to only allow the NTEF branch. Seeing as none of the afromentioned are able to be NTEF, nothing horrible should go wrong.
I figured the other `#define`s are also meant in a broad sense.

**Comment by JohnBigless** [comment]
Oh wow, Travis didn't actually fuck up.

File: maps/torch/loadout/_defines.dm [hunk 3]
```diff
@@ -51,6 +51,9 @@
 //For members of the command and command support department. Why wasn't this here before?
 #define COMMAND_ROLES list(/datum/job/captain, /datum/job/hop, /datum/job/rd, /datum/job/cmo, /datum/job/chief_engineer, /datum/job/hos, /datum/job/sea, /datum/job/sea/marine, /datum/job/bridgeofficer, /datum/job/liaison, /datum/job/bodyguard, /datum/job/bailiff, /datum/job/psiadvisor, /datum/job/adjudicator)
 
+//For jobs that have at least O-1 in either NTEF or SMC, but aren't considered Command or Command Support.
+#define MISCOFFICER_ROLES list(/datum/job/scientist, /datum/job/qm, /datum/job/chaplain, /datum/job/psychiatrist, /datum/job/senior_doctor, /datum/job/pathfinder)
```
Comment (Roland410) [inline maps/torch/loadout/_defines.dm:55]:
```suggestion
#define OFFICER_ROLES list(/datum/job/scientist, /datum/job/qm, /datum/job/chaplain, /datum/job/psychiatrist, /datum/job/senior_doctor, /datum/job/pathfinder)
```

Comment (JohnBigless) [reply maps/torch/loadout/_defines.dm:55]: k

**Comment by JohnBigless** [comment]
http://prntscr.com/t52y85
Nice.

**Comment by JohnBigless** [comment]
image

**Pull request merged**